tests/dummy_afu/test_dummy_afu.cpp

```diff
@@ -124,6 +159,51 @@ TEST_P(dummy_afu_p, main_mmio) {
   EXPECT_EQ(0, app_->main(args_.size(), const_cast<char**>(args_.data())));
 }
 
+/*
+ * @test       main_mmio_count
+ * @brief      Test: test main with mmio subcommand and count > 1
+ * @details    Test the main entry point for dummy_afu
+ */
+TEST_P(dummy_afu_p, main_mmio_count) {
```

Comment:
Is something calling free() on all these strdup'ed strings?

Reply:
Yes, line 131.
https://github.com/OPAE/opae-sdk/blob/ded0b4602b6c014542de6297a6783a2ad6d14db8/tests/dummy_afu/test_dummy_afu.cpp#L121-L133